packages/compiler/src/plugins.js

```diff
@@ -60,10 +61,27 @@ export const processMyst = () => {
           // a note! we want to replace the code chunk with a svelte component
           let newNode = {
             type: "html",
-            value: `<Admonition type={"${mystType}"}>${node.value}</Admonition>`,
+            value: `<Admonition type={"${mystType}"}>${micromark(
+              node.value
+            )}</Admonition>`,
           };
           parent.children[index] = newNode;
           return index;
+        } else if (mystType ==="panels") {
+          let cards = parsePanels(node.value);
+          // parse each card
+          let newNode = {
+            type: "html",
+            value: `<Panels>
+                    <ul>
+                    {#each ${cards} as card}
```

Comment:
You'll have to embed this expression within a single string-- the `cards` expression won't transfer into the generated svelte component. It may be cleaner to do this with a mustache template, which we use in some other parts of the codebase. e.g. https://github.com/irydium/irydium/blob/1e7c42a55bc0632ef5bf841176fd8b10bb280893/packages/compiler/src/svelteToHTML.ts#L134